app/views/tasks/_new_tasks.html.erb

```diff
@@ -12,7 +12,7 @@
       <% tasks.each do |task| %>
           <ul class="list-group">
             <li class="list-group-item">
-              <%= task.name %>
+              <%= task.name %>(<%= link_to (task.user.initials), profile_path(task.user.profile) %>)
```

Comment:
the same as [above](https://github.com/dKotetunov/TasksManager/pull/11/files#r28692964)